src/axom/mint/mesh/internal/ConnectivityArray_typed_indirection.hpp

```diff
@@ -402,13 +361,22 @@ class ConnectivityArray<TYPED_INDIRECTION>
    */
   bool isInSidre() const
   {
+#ifdef AXOM_MINT_USE_SIDRE
     bool consistent = true;
-    bool is_in_sidre = m_values->isInSidre();
-    consistent &= is_in_sidre == m_offsets->isInSidre();
-    consistent &= is_in_sidre == m_types->isInSidre();
+    auto& values_ref = *m_values;
+    auto& offsets_ref = *m_offsets;
+    auto& types_ref = *m_types;
+    bool is_in_sidre = typeid(values_ref) == typeid(sidre::Array<IndexType, 1>);
```

Comment:
gut check: Is this use of `typeid` portable across all compilers we care about?

Reply:
`typeid` in the way we're using it here requires RTTI, which should be enabled by default on our set of compilers. I think as an alternative, we could just create/set an enum value on construction for where the memory is, if that would be clearer.

Reply:
Thanks. I'm ok with the current solution if it works. 

Do we have sufficient tests to ensure that we'd know if it breaks? If not, we should probably add them.

Reply:
I ended up removing this -- while the use of `typeid()` will error at compile-time with gcc/clang's `-fno-rtti`, it will just silently produce invalid results for polymorphic values on MSVC if compiled without RTTI support.

Reply:
@kanye-quest , thank you for checking that.